bin/node/bench/src/tempdb.rs

```diff
@@ -41,17 +41,14 @@ impl KeyValueDB for ParityDbWrapper {
 	}
 
 	/// Write a transaction of changes to the buffer.
-	fn write_buffered(&self, transaction: DBTransaction) {
+	fn write(&self, transaction: DBTransaction) -> io::Result<()> {
 		self.0.commit(
 			transaction.ops.iter().map(|op| match op {
 				kvdb::DBOp::Insert { col, key, value } => (*col as u8, &key[key.len() - 32..], Some(value.to_vec())),
 				kvdb::DBOp::Delete { col, key } => (*col as u8, &key[key.len() - 32..], None),
+				kvdb::DBOp::DeletePrefix { col: _, prefix: _ } => unimplemented!()
```

Comment:
Why don't we need to implement this?

Reply:
tbh, I'm unfamiliar with parity-db, however i don't think it supports prefix-based retrieval

Reply:
Ahh that is `ParityDB`, so nevermind ;)